### ISSUE TITLE
write handlers for NELM and NSW values and handling other (exceptional?) situations where the default settings fail

### ISSUE THREAD

**Issue by QuantumChemist** [autoatml/autoplex#287]
maybe we can also think of implementing this in custodian directly?

**Comment by JaGeo** [comment]
Yes. But we can draft them here first and then move them if we think they are useful

**Comment by QuantumChemist** [comment]
sounds like a good idea 😃

**Comment by JaGeo** [comment]
Can we also check if we can inherit some changes through the workflow. E.g., we do a relaxation first and then use the sigma from there everywhere
